charts/loki-distributed/templates/memcached-chunks/statefulset-memcached-chunks.yaml

```diff
@@ -147,4 +153,8 @@ spec:
           requests:
             storage: {{ .Values.memcachedChunks.persistence.size | quote }}
   {{- end }}
+  {{- with .Values.memcachedChunks.volumeClaimTemplates }}
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+{{- end }}
```

Comment:
I can't place this last `end`

Reply:
The last `end` is for `{{- if .Values.memcachedChunks.enabled }}` at the top.

https://github.com/jdgeisler/grafana-helm-charts/blob/a9587fbeef83888a2858d5a4f737078ff1cc5db0/charts/loki-distributed/templates/memcached-chunks/statefulset-memcached-chunks.yaml#L1